### PR TITLE
fix(addon-commerce): fix problem with `InputCardGrouped` in safari

### LINKED ISSUE #1679: 🐞 - credit cards not showing up in safari
### Which @taiga-ui/* package(s) are the source of the bug?

core

### Please provide a link to a minimal reproduction of the bug

https://taiga-ui.dev/components/input-card-grouped

### Description

image

### Angular version

latest

### Taiga UI version

latest

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #1679

https://user-images.githubusercontent.com/12021443/165838470-cd31e93d-7259-46d8-aef2-a223bf0e687a.mov

## What is the new behavior?

https://user-images.githubusercontent.com/12021443/165838673-b1a77d90-f426-49de-a346-343111739468.mov

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/demo-integrations/cypress/tests/addon-commerce/input-card-grouped.spec.ts [hunk 1]
```diff
@@ -19,4 +18,85 @@ describe('InputCardGrouped', () => {
         cy.get('@wrapper').click({force: true});
         cy.matchImageSnapshot('03-input-card-grouped-unfocused', {capture: 'viewport'});
     });
+
+    it('set value and disable', () => {
+        cy.tuiVisit('components/input-card-grouped/API?tuiMode=null');
+        cy.get('#demoContent').as('wrapper');
```
Comment (nsbarsukov) [inline projects/demo-integrations/cypress/tests/addon-commerce/input-card-grouped.spec.ts:24]:
```suggestion
        cy.get('#demoContent').should('be.visible').as('wrapper');
```

File: projects/demo-integrations/cypress/tests/addon-commerce/input-card-grouped.spec.ts [hunk 2]
```diff
@@ -19,4 +18,85 @@ describe('InputCardGrouped', () => {
...
+            .findByAutomationId('tui-input-card-grouped__card')
+            .type('1234 4567 1000 1112')
+            .wait(5000); // waiting before closing toast for skip flaky test
+
+        cy.get('.t-table tr')
+            .eq(1) // formControl.disable()
+            .findByAutomationId('tui-toggle__checkbox')
+            .click({force: true});
+
+        cy.matchImageSnapshot('04-input-card-grouped-disabled', {capture: 'viewport'});
```
Comment (nsbarsukov) [inline projects/demo-integrations/cypress/tests/addon-commerce/input-card-grouped.spec.ts:36]: Do we actually need to take screenshot of the whole page ?

It is so time-expensive(

File: projects/demo-integrations/cypress/tests/addon-commerce/input-card-grouped.spec.ts (hunk 1 above)
Comment (splincode) [reply projects/demo-integrations/cypress/tests/addon-commerce/input-card-grouped.spec.ts]: fixed

File: projects/addon-commerce/components/input-card-grouped/input-card-grouped.template.html [hunk 3]
```diff
@@ -26,7 +26,7 @@
                 tuiInputMode="numeric"
                 class="t-input t-input_card"
                 [class.t-input_inert]="cardPrefilled"
-                [class.t-input_hidden]="isCardCollapsed"
+                [class.t-input_empty]="card.length < 1 && focused"
```
Comment (vladimirpotekhin) [inline projects/addon-commerce/components/input-card-grouped/input-card-grouped.template.html:29]: nit: how do you like `!card.length && focused`  ?

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/addon-commerce/components/input-card-grouped/input-card-grouped.template.html (hunk 3 above)
Comment (splincode) [reply projects/addon-commerce/components/input-card-grouped/input-card-grouped.template.html]: fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**